drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/handlers/CommonOnMoveHandlerTest.java

```diff
@@ -0,0 +1,316 @@
+import org.drools.scenariosimulation.api.model.FactMappingValue;
+import org.drools.scenariosimulation.api.model.FactMappingValueStatus;
+import org.drools.scenariosimulation.api.model.Scenario;
+import org.drools.workbench.screens.scenariosimulation.client.popover.ErrorReportPopoverPresenter;
+import org.drools.workbench.screens.scenariosimulation.client.popover.PopoverView;
+import org.drools.workbench.screens.scenariosimulation.client.resources.i18n.ScenarioSimulationEditorConstants;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.uberfire.ext.wires.core.grids.client.model.GridColumn;
+import org.uberfire.mvp.Command;
+
+import static org.drools.workbench.screens.scenariosimulation.client.TestProperties.CELL_WIDTH;
+import static org.drools.workbench.screens.scenariosimulation.client.TestProperties.COLUMN_INDEX;
+import static org.drools.workbench.screens.scenariosimulation.client.TestProperties.DX;
+import static org.drools.workbench.screens.scenariosimulation.client.TestProperties.DY;
+import static org.drools.workbench.screens.scenariosimulation.client.TestProperties.ERROR_VALUE;
+import static org.drools.workbench.screens.scenariosimulation.client.TestProperties.EXCEPTION;
+import static org.drools.workbench.screens.scenariosimulation.client.TestProperties.LARGE_LAYER;
+import static org.drools.workbench.screens.scenariosimulation.client.TestProperties.MX;
+import static org.drools.workbench.screens.scenariosimulation.client.TestProperties.MY;
+import static org.drools.workbench.screens.scenariosimulation.client.TestProperties.NULL;
+import static org.drools.workbench.screens.scenariosimulation.client.TestProperties.RAW_VALUE;
+import static org.drools.workbench.screens.scenariosimulation.client.TestProperties.ROW_INDEX;
+import static org.drools.workbench.screens.scenariosimulation.client.TestProperties.SCROLL_LEFT;
+import static org.drools.workbench.screens.scenariosimulation.client.TestProperties.SCROLL_TOP;
+import static org.drools.workbench.screens.scenariosimulation.client.TestProperties.TINY_LAYER;
+import static org.junit.Assert.assertFalse;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Matchers.isA;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(LienzoMockitoTestRunner.class)
+public class CommonOnMoveHandlerTest extends AbstractScenarioSimulationGridHandlerTest {
+
+    private CommonOnMoveHandler commonOnMoveHandler;
+
+    @Mock
+    private ErrorReportPopoverPresenter errorReportPopupPresenterMock;
+    @Mock
+    private Scenario scenarioMock;
+    @Mock
+    private FactMappingValue factMappingValueMock;
+    @Mock
+    private AbsolutePanel scrollPanel;
+    @Mock
+    private Element element;
```

Comment:
To be consistent, please use name `elementMock`

Reply:
Done!